### PR TITLE
Update gitops new dd

### PR DESCRIPTION
**Pull request by mbkranz**
Link to Jira ticket if there is one:

### Environments
QA

### Description of changes
Updated gitops with new DD. Also deleted the Data tab for now as it is not used (but another tab will be added with the time_point ETL mapping).

### CODE REVIEW

File: qa-jcoin.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -708,12 +626,5 @@
     }
   ],
   "showArboristAuthzOnProfile": true,
-  "showFenceAuthzOnProfile": false,
-  "componentToResourceMapping": {
-    "HUB Documentation": {
```
Comment (paulineribeyre) [inline qa-jcoin.planx-pla.net/portal/gitops.json:713]: are you removing this on purpose?

Comment (mbkranz) [reply qa-jcoin.planx-pla.net/portal/gitops.json]: I did not - thanks for catching - I added this back in

Comment (mbkranz) [reply qa-jcoin.planx-pla.net/portal/gitops.json]: I did mean to remove the cmc_data_type to make it consistent with PROD

**Review by paulineribeyre — APPROVED** [review]

**Comment by paulineribeyre** [comment]
deploying to QA now

**Pull request merged**